test/built-ins/Temporal/PlainDateTime/prototype/toString/calendarname-auto.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.tostring
+description: Possibly display calendar when calendarName is "auto"
+features: [Temporal]
+---*/
+
+
+const dt = new Temporal.PlainDateTime(1976, 11, 18, 15, 23);
+const cal = {
+  toString() { return "bogus"; }
+};
+const expected = "1976-11-18T15:23:00";
+
+assert.sameValue(dt.toString(), expected, "default is calendar = auto (zero arguments)");
+assert.sameValue(dt.toString({ calendarName: "auto" }), expected, "shows only non-ISO calendar if calendarName = auto");
+
+assert.sameValue(
+  dt.withCalendar("iso8601").toString({ calendarName: "auto" }),
+  expected,
+  "Don't show ISO calendar even if calendarName = auto"
+);
+
+assert.sameValue(
+  dt.withCalendar(cal).toString({ calendarName: "auto" }),
+  "1976-11-18T15:23:00[u-ca=bogus]",
+  "Show non-ISO calendar if calendarName = auto"
+);
+
+const dt2 = new Temporal.PlainDateTime(1976, 11, 18, 15, 23, cal);
+
+assert.sameValue(
+  dt2.toString({ calendarName: "auto" }),
+  expected,
+  "Don't show calendar if calendarName = auto & PlainDateTime has non-ISO calendar"
+);
```

Comment:
This should be the same as the check right above, but it isn't because you're passing `cal` for the seconds argument.

Reply:
Quite right, oops. I'll update this.